js/viz/docs/docBaseWidget.js

```diff
@@ -535,7 +535,13 @@ var BaseWidget = {
             * @default 2
             */
             blur: 2
-        }
+        },
+        /**
+        * @name BaseWidgetOptions.tooltip.cornerRadius
+        * @type number
+        * @default 0
+        * @default 4 @for Material
+        */
```

Comment:
@RomanTsukanov @IlyaKhD 
is this correct description for default value?

Reply:
It's OK